manim/mobject/types/vectorized_mobject.py

```diff
@@ -1213,6 +1213,62 @@ def __setitem__(self, key, value):
             self.remove(key)
         self.add([(key, value)])
 
+    def __delitem__(self, key):
+        """Override the del operator for deleting an item.
+
+        Parameters
+        ----------
+        key : Hashable
```

Comment:
Maybe you should typing here? 
typing.Hashable?

Reply:
@PgBiel what do you recommend?

Reply:
@naveen521kk do you think this is blocking? If not, could you please approve so that we can merge this asap?

Reply:
Use 
```suggestion
        key : :class:`typing.Hashable`
```
same for all the classes where you use Hashable.

Reply:
Done in the last commit!